site/data/refDataApi.ts

```diff
@@ -45,6 +45,7 @@ export const fetchStops = async (input: FetchStopsInput) => {
 interface FetchServicesInput {
     adminAreaCodes?: string[];
     dataSource?: Modes;
```

Comment:
could we rename this interface to DataSource

Reply:
done